pkgs/development/python-modules/wfuzz/default.nix

```
@@ -0,0 +1,39 @@
+{ buildPythonPackage
+, chardet
+, configparser
+, fetchPypi
+, future
+, lib
+, pyparsing
+, pycurl
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "wfuzz";
+  version = "2.4.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19rkd2qdbcf16rgqds5fxvwlyxcmgw70ljzh443p26gmfknh13rn";
+  };
+
+  buildInputs = [
+    pyparsing
+    configparser
+  ];
+
+  propagatedBuildInputs = [
+    chardet
+    future
+    pycurl
+    six
+  ];
+
```

looks like tests aren't ran, please add by checking out from github:
```suggestion
  # taken from .travis.yml
  checkPhase = ''
    ${python.interpreter} -m unittest discover tests/
  ''
```

---

currently:
```
writing manifest file 'wfuzz.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```

---

Ah, missed that, thanks.

However, the tests require `docker` and `docker-compose`: https://github.com/xmendez/wfuzz/blob/master/.travis.yml#L12 ...
Should they be run in that case?

---

avoid the tests which require the server

---

```
  checkInputs = [ pytest netaddr ];
  checkPhase = ''
    HOME=$TMPDIR pytest \
      tests/test_{moduleman,filterintro,reqresp,api,clparser,dotdict}.py
  '';
```